tsl/test/sql/continuous_aggs_permissions.sql

```diff
@@ -164,3 +186,33 @@ CALL refresh_continuous_aggregate('mat_perm_view_test', NULL, NULL);
 
 --but the old data will still be there
 SELECT * FROM mat_perm_view_test;
+
+\set VERBOSITY default
+
+\c :TEST_DBNAME :ROLE_DEFAULT_PERM_USER
```

Comment:
Would be good with some comments describing what is tested and what the expected outcome is. Now one has to read the SQL code to try to figure it out.

Reply:
Added a comment. It can be even clearer if we single out the user that we change the grants for, but that seems excessive at this point.